pcdsdevices/targets.py

```diff
@@ -125,9 +132,9 @@ class XYTargetGrid():
                           x_spacing=1.0, y_spacing=1.0, x_comp=0.05,
                           y_comp=0.01)
     """
-    def __init__(self, x=None, y=None, x_init=None, x_spacing=None,
-                 x_comp=0.0, y_init=None, y_spacing=None, y_comp=0.0,
-                 name=None):
+
+    def __init__(self, name, x=None, y=None, x_init=None, x_spacing=None,
+                 x_comp=0.0, y_init=None, y_spacing=None, y_comp=0.0):
```

Comment:
Just being a little cautious- any worries about changing the signature here?
Previously, `name` was an optional kwarg
Now, `name` is a required arg (positional  or keyword okay)
Usually in ophyd, `name` is a required keyword-only kwarg (which is neither the previous nor the updated situation here)

Reply:
@slactjohnson This `XYTargetGrid` is being used in `mec`, correct? Should double-check that a signature update isn't immediately breaking

Reply:
I went back to the original version in 393ee51 - I am still using `StackStage` - just so we can give the option to the user to insert either a motor object or a PV, but I am wondering if I should not use `StackStage` at all?

Reply:
@ZLLentz I believe we have `XYTargetGrid` instantiated for MEC, but I don't know how much they use it (even though they asked for it). Based on the comments below, it looks like the change in signature may have been avoided anyway?

Reply:
Yes, it turns out we didn't need to do this, which is great. Changing API can be a hassle when it needs to be done.